plugins/Diagnostics/Diagnostic/DbMaxPacket.php

```diff
@@ -43,7 +43,9 @@ public function execute()
             $formatter = new Formatter\Html();
             $pretty = $formatter->getPrettySizeFromBytes($maxPacketBytes['Value'], 'M', $precision = 1);
             $configured = str_replace(array(' M', '&nbsp;M'), 'MB', $pretty);
-            $comment = Piwik::translate('Diagnostics_MysqlMaxPacketSizeWarning', array('64MB', $configured));
+            $comment = Piwik::translate('Diagnostics_MysqlMaxPacketSizeWarning',
+                ['<a href="https://dev.mysql.com/doc/refman/8.0/en/packet-too-large.html" rel="noreferrer noopener" target="_blank">',
```

Comment:
We can use version agnostic link here and it redirects to the latest one automatically
```suggestion
                ['<a href="https://dev.mysql.com/doc/refman/en/packet-too-large.html" rel="noreferrer noopener" target="_blank">',
```

Reply:
Nice! :+1: